solr/CHANGES.txt

```diff
@@ -85,7 +85,9 @@ Improvements
   `GET /api/node/logging/messages`.  And the threshold for the log "listener" can be modified with
   `PUT /api/node/logging/messages/threshold`.  (Jason Gerlowski, Calvince Otieno)
 
-* SOLR-16504: Convert CLI tools to use Jetty HTTP 2 client.  (Bence Szabo via Eric Pugh)  
+* SOLR-16504: Convert CLI tools to use Jetty HTTP 2 client.  (Bence Szabo via Eric Pugh)
```

Comment:
line 88 should result unchanged

Reply:
oh ok... I have also seen that this file was recently modified at the same point and our fork doesn't contain the change